src/cli/README_SRP.md

```diff
@@ -0,0 +1,181 @@
+# OpenThread CLI - SRP (Service Registration Protocol)
+
+## Quick Start
+
+### Build with SRP Server & Client support
+
+Use the `SRP_SERVER=1 SERVICE=1 ECDSA=1` build switches to enable SRP Server API support, and the `SRP_CLIENT=1 ECDSA=1` build switches for SRP Client API support.
+
+Make simulation SRP Server & Client:
+
+```bash
+./bootstrap
+make -f examples/Makefile-simulation SRP_SERVER=1 SERVICE=1 SRP_CLIENT=1 ECDSA=1
+```
+
+## Simulation Example
+
+### Start SRP Server
+
+Start the SRP Server node:
+
+```bash
+./output/simulation/bin/ot-cli-ftd 1
+```
+
+Setup a Thread network and start the SRP Server:
+
+```bash
+> panid 0xface
```

Comment:
Use dataset command when forming the network - see [src/cli/README_DATASET.md#form-network](https://github.com/openthread/openthread/blob/master/src/cli/README_DATASET.md#form-network)